constitution.tex

```diff
@@ -544,7 +545,8 @@
 
 \asubsubsubsection{Introductory Evaluation}
 The Introductory Evaluation is the process by which House chooses which Introductory Members to extend an offer of full Resident or Non-Resident Membership.
-Occurs between the sixth and tenth week of each semester.
+Occurs during the sixth week of each semester. If the evaluations director chooses to hold a second Introductory Process, the Introductory Evaluation will also occur during the twelfth week of each semester.
```

Comment:
```suggestion
Occurs at the conclusion of the sixth week of the Process.
```
Since the process may start the first or second week, maybe this should be tied to the length of the process, rather than the date?

Reply:
Also, each sentence gets its own line